lib/mix/test/mix/umbrella_test.exs

```diff
@@ -384,21 +414,26 @@ defmodule Mix.UmbrellaTest do
           def foo(list), do: list
         end
         """)
+
         Mix.Task.run("compile")
         assert File.regular?("_build/dev/lib/bar/consolidated/Elixir.Foo.beam")
         assert Mix.Tasks.Compile.Protocols.run([]) == :noop
 
         # Mark protocol as outdated
-        File.touch!("_build/dev/lib/bar/consolidated/Elixir.Foo.beam",
-                    {{2010, 1, 1}, {0, 0, 0}})
+        File.touch!("_build/dev/lib/bar/consolidated/Elixir.Foo.beam", {{2010, 1, 1}, {0, 0, 0}})
+
+        ensure_touched(
+          "_build/dev/lib/foo/ebin/Elixir.Foo.beam",
+          File.stat!("_build/dev/lib/bar/consolidated/.compile.protocols").mtime
```

Comment:
Can you please move the computation of the mtime to the previous line? Something like this:

```
mtime = File.stat!("_build/dev/lib/bar/consolidated/.compile.protocols").mtime
ensure_touched("_build/dev/lib/foo/ebin/Elixir.Foo.beam", mtime)
```

Similar changes should be done through this file, thank you!